src/js/control-bar/text-track-controls/subs-caps-button.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @file tracks-button.js
```

Comment:
Just noticed - this file name needs updating. (<Captain Pedantic, at you're service ;-)>)